distiller/settings/development.py

```diff
@@ -10,3 +10,5 @@
 DEBUG = True
 
 SECRET_KEY = 'SECRET'
+
+LOAD_TABLE_ROOT = str(PROJECT_ROOT / 'imports')
```

Comment:
`os.path.join`? Actually, not sure what this is for.

Reply:
This is a default download dir, if you run the download task locally. `PROJECT_ROOT` is a `pathlib.Path` object so `os.path.join` isn't required.